src/Parser/Lexer.mll

```diff
@@ -38,6 +39,44 @@ let kw_map =
   ; "_",        UNDERSCORE
   ] |> List.to_seq |> Hashtbl.of_seq
 
+(* let op_map = Hashtbl.create 32 *)
+let op_map = 
+  let open YaccParser in
+  [ "->", ARROW
+  ; "→",  ARROW
```

Comment:
I think we should get rid of unicode arrows instead of moving them to general lexer rules. The reason is that position tracking in ocamllex does not support unicode.